.github/workflows/pypi-upload.yml

```
@@ -0,0 +1,46 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+name: Upload the cocotb package to PyPi
```

Kind of a long name, gets chopped off on the Actions page.
```suggestion
name: Publish to PyPi
```